doc/storageprofile.md

```diff
@@ -36,19 +36,28 @@ status:
     cloneStrategy: snapshot
 ```
 
-Current version supports the following parameters:
+### Parameters
 - `cloneStrategy` - defines the preferred method for performing a CDI clone
 - `claimPropertySets` contains a list of `claimPropertySet`
   - `accessMode` - contains the desired access modes the volume should have
-  - `volumeMode` - defines what type of volume is required by the claim
+  - `volumeMode` - defines what type of volume is required by the claim  
+  These are ordered by preference, so some things to note are:  
+      Usually Block is preferred over Filesystem for performance reasons (fewer layers)  
+      RWX over RWO (live migration support)
+- `dataImportCronSourceFormat` DataImportCron (recurring polling of golden registry sources) was originally designed to only maintain PVC sources, However, for certain storage types, we know that snapshots sources scale better. Some details and examples can be found in [clone-from-volumesnapshot-source](./clone-from-volumesnapshot-source.md).
 
 Values for accessModes and volumeMode are exactly the same as for PVC: `accessModes` is a list of `[ReadWriteMany|ReadWriteOnce|ReadOnlyMany]`
```

Comment:
Maybe explain that we are explicitly not interested in ReadWriteOncePod. We are aware that it exists but it makes certain things more difficult and it doesn't really help us in our use cases.

Reply:
actually, we should play nice for the most part. context in https://youtu.be/T5l_WqLpAeQ?t=2385

Reply:
anyway, addressed